Bottom: 02461cfc9c74727290e819ca4332d0fc5d50ccf3
Top:    0c85af868f0bded0ac09890ebd9a7cd8a5361bfc
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-12-29 10:41:16 +0800

Refresh of tuntap-rename-struct-0

---

diff --git a/drivers/net/tun.c b/drivers/net/tun.c
index d29fa81..f7d6708 100644
--- a/drivers/net/tun.c
+++ b/drivers/net/tun.c
@@ -916,7 +916,6 @@ static int run_filter(struct tap_filter *filter, const struct sk_buff *skb)
 }
 
 
-
 /*
  * Checks whether the packet is accepted or not.
  * Returns: 0 - drop, !=0 - accept
